ert_shared/dark_storage/enkf.py

```diff
@@ -24,7 +24,7 @@ def init_facade() -> None:
 
     _config = ResConfig(configfile)
     os.chdir(_config.config_path)
-    _ert = EnKFMain(_config, strict=True)
+    _ert = EnKFMain(_config, strict=True, read_only=True)
```

Comment:
Not sure if it is possible - but would it have been better having default as `read_only=True` and that you need to opt in to be able to write?

Reply:
I agree that would be better but it is not really possible without a significant refactor. I can add an issue for this and we can do the refactor at a later time. What do you think?

Reply:
I think that sounds fine! Just have an issue labeled with `improvement` or equivalent would be sufficient. It is not critical - but should be part of refactoring whenever it fits our schedule.

Reply:
I added https://github.com/equinor/ert/issues/2775